dynamics-nav/SETRECORD-Function--Page-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETRECORD Function (Page)"
+description: SETRECORD Function (Page) Sets the current record for the page. This topic contains information on the SETRECORD Function (Page) syntax, parameters, remarks, and examples.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.